pkg/services/services.go

```diff
@@ -223,7 +223,9 @@ func klogVerbosityArg(debug bool) string {
 func getHostnameCmd(platformType config.PlatformType) string {
 	switch platformType {
 	case config.AWSPlatformType:
-		return "Get-EC2InstanceMetadata -Category LocalHostname"
+		// For the instance metadata service to work on BYOH Windows instances using a custom Windows AMI, the AMI must
+		// be a standardized image created using the AWS's sysprep workflow. https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/Creating_EBSbacked_WinAMI.html#ami-create-standard
```

Comment:
What does the doc have to do with the metadata endpoint?

Reply:
Please add the doc for the metadata endpoint here.

Reply:
In addition are using IMDS v1 or v2?

Reply:
> What does the doc have to do with the metadata endpoint?

Nothing. Not sure where that came from 👽